tests/Doctrine/Tests/ORM/Functional/ValueObjectsTest.php

```diff
@@ -242,6 +243,48 @@ public function testThrowsExceptionOnInfiniteEmbeddableNesting($embeddableClassN
         ));
     }
 
+    public function testCRUDOfNullableEmbedded()
```

Comment:
@lcobucci consider splitting this test into smaller tests depending on each other via `@depends`